cumulusci/cli/cci.py

```diff
@@ -65,13 +65,16 @@ def timestamp_file():
         yield f
 
 
-def get_latest_version():
+def get_latest_final_version():
     """ return the latest version of cumulusci in pypi, be defensive """
```

Comment:
during the refactor the "be defensive" got moved into the calling method. update docstring plz